src/app/app.routes.ts

```diff
@@ -37,6 +37,7 @@ export const routes: RouterConfig = [
       { path: 'appointments', component: AppointmentAdminComponent },
       { path: 'appointments/new', component: AppointmentFormComponent },
       { path: 'appointments/:id', component: AppointmentFormComponent },
+      { path: 'appointments/remove/:id', component: AppointmentFormComponent },
```

Comment:
It is [here](https://github.com/Sirimangalo/meditation-plus-server/pull/19/files#diff-e3f6107adee860929b706ea0a7b7f2d6R177) and [here](https://github.com/Sirimangalo/meditation-plus-angular/blob/0b32e07d4c72a91394d3368b3e59dbedbcc201f8/src/app/appointment/appointment.service.ts#L70)??